parsec/backend/postgresql/organization.py

```diff
@@ -197,8 +197,30 @@ async def stats(self, id: OrganizationID) -> OrganizationStats:
         async with self.dbh.pool.acquire() as conn, conn.transaction():
             await self._get(conn, id)  # Check organization exists
             result = await conn.fetchrow(*_q_get_stats(organization_id=id))
+            users = 0
+            active_users = 0
+            users_per_profile_detail = {p: {"active": 0, "revoked": 0} for p in UserProfile}
+            for u in result["users"]:
+                revoked_on, profile = u
+                users += 1
+                if revoked_on:
+                    users_per_profile_detail[UserProfile[profile]]["revoked"] += 1
```

Comment:
I didn't know you could do `UserProfile[profile]` and use a `str` as key, this is really cool 😃 

That mean all the `STR_TO_xxx` can be removed \o/
https://github.com/Scille/parsec-cloud/blob/99d94710cf868dafabdc106a830991e05d62e345/parsec/backend/postgresql/utils.py#L17-L24